README.md

```diff
@@ -279,14 +279,15 @@ The Subject has a single key.
 <br> 
 
 ![Figure 3_TX Anatomy](https://github.com/user-attachments/assets/b00ac2d0-3a3d-4b06-aac6-f9a529477d0a)
+-
```

Comment:
The multisignature unlocking script needs to be corrected:

`OP_0 SigC0 SigS0` is the correct pattern. There is a null data (bug from years ago) and a list of signatures. There is no need to push the pubkeys as they are already in the locking script.